src/test/java/teammates/test/driver/GaeSimulation.java

```diff
@@ -123,10 +123,24 @@ public void loginAsAdmin(String userId) {
         helper.setEnvIsAdmin(true);
     }
 
+    /**
+     * Logs in the user to the GAE simulation environment as an unregistered user
+     * (without any right).
+     */
+    @Deprecated
+    public void loginAsUnregistered(String userId) {
```

Comment:
What is the standard way of login using GAE now?

Reply:
The same. The reason I mark this as deprecated is because the method contains assertions, which I don't really like (this is a test driver which should not have any business with assertion).